BeeKit/Model/DataPoint.swift

```diff
@@ -78,6 +78,7 @@ public class DataPoint: NSManagedObject {
         value = json["value"].numberValue
         comment = json["comment"].stringValue
         requestid = json["requestid"].stringValue
+        updatedAt = json["updated_at"].intValue
```

Comment:
How is updated_at different than last_touch?

I see updated_at for a goal at api.beeminder.com

> updated_at (number): Unix timestamp of the last time this goal was updated

and do not see last_touch mentioned there

Reply:
> lasttouch is meant to be more conservative. so it's the timestamp that the goal was touched in a "meaningful" way: data added or graph regenerated or... 
> updated_at is any time the goal was saved, including for reasons the user wouldn't care about